src/css-modules/CategoryTiles.module.css

```diff
@@ -32,23 +31,24 @@
 }
 
 .tile:focus .icon {
-  filter: invert(100%) sepia(100%) saturate(0%) hue-rotate(109deg) brightness(102%) contrast(102%);
+  filter: invert(100%) sepia(100%) saturate(0%) hue-rotate(109deg)
+    brightness(102%) contrast(102%);
 }
 
 .icon {
   margin: 10px;
 
   height: 24px;
   width: 24px;
-  filter: invert(42%) sepia(8%) saturate(1531%) hue-rotate(92deg) brightness(97%) contrast(92%);
+  filter: invert(42%) sepia(8%) saturate(1531%) hue-rotate(92deg)
+    brightness(97%) contrast(92%);
 }
 
 .title {
   display: flex;
   align-items: center;
   justify-content: center;
   font-size: 10px;
-  margin-bottom: 8px;
   font-weight: 600;
-  letter-spacing: 1.3px;
```

Comment:
I think it looks better with letter spacing

Reply:
I know but with letter spacing it looks more like the figma file. You can compare and check